src/audits/manifest/short-name-length.js

```diff
@@ -48,16 +49,26 @@ class ManifestShortNameLength extends Audit {
   static audit(artifacts) {
     let isShortNameShortEnough = false;
     let debugString;
+    let resolvedName;
     const manifest = artifacts.manifest.value;
     const suggestedLength = 12;
 
-    if (manifest && manifest.short_name && manifest.short_name.value) {
+    if (manifest) {
+      const shortNameValue = _get(manifest, 'short_name.value');
+      const nameValue = _get(manifest, 'name.value');
+
+      // When no shortname can be found we look for a name
+      // https://developer.chrome.com/apps/manifest/name#short_name
+      resolvedName = shortNameValue || nameValue;
```

Comment:
let's be explicit here and check for !== undefined instead of depending on the type coercion